python/spacewalk/satellite_tools/repo_plugins/yum_src.py

```diff
@@ -525,6 +526,21 @@ def _get_mirror_list(self, repo, url):
         mirrorlist_path = os.path.join(repo.root, 'mirrorlist.txt')
         returnlist = []
         content = []
+        # If page not plaintext or xml, is not a valid mirrorlist or metalink,
+        # so continue without it.
+        proxies = get_proxies(self.proxy_url, self.proxy_user, self.proxy_pass)
+        cert = (self.sslclientcert, self.sslclientkey)
+        verify = self.sslcacert
+        try:
+            webpage = requests.get(url, proxies=proxies, cert=cert, verify=verify)
+            content_type = webpage.headers["Content-Type"]
+            if "text/plain" not in content_type and "xml" not in content_type:
```

Comment:
maybe change it to somethinkg like:
```
if webpage.headers["Content-Type"] not in ("text/plain", "xml"...)
```
Just a tuple with the exact possible values for `Content-Type`.

Reply:
Oh, sorry, just checked the behaviour; I did it this way because the content_type is output in the form:
"text/plain; charset=UTF-8"
so it did need to be part of the output rather than the whole thing! I suppose it could split on the semicolon, but I think that would overcomplicate it, so unless there are times when it's going to include 'xml' or 'text/plain' but be the wrong type of document, I think it's better to leave it.